recipe/meta.yaml

```diff
@@ -92,7 +92,13 @@ requirements:
     - m2-git    # [win]
     - libgomp   # [linux]
     - llvm-openmp    # [osx]
-    - cmake
+    # hmaarrfk - 2024/08 -- I feel like the patch added in
+    # https://gitlab.kitware.com/cmake/cmake/-/commit/df551ee53875bbfc888d92c7c523c9b6b2154283#
+    # is wrong for conda-forge....
+    # https://github.com/numpy/numpy/issues/23775
+    # https://gitlab.kitware.com/cmake/cmake/-/blob/v3.30.2/Modules/FindPython/Support.cmake?ref_type=tags#L3906
+    - cmake !=3.30.0,!=3.30.1,!=3.30.2        # [osx and blas_impl == "mkl"]
```

Comment:
This should be fixed as of 3.30.2 AFAICT: https://github.com/Kitware/CMake/commit/df551ee53875bbfc888d92c7c523c9b6b2154283

Reply:
What aspect of that patch is wrong in conda-forge IYO (I'm not denying that detection didn't work before of course)?

Reply:
i couldn't figure it out either tbh.

Reply:
see: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/253

Reply:
lets discuss and try to troubleshoot there.